osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/monitoring/GetPowerQualityProfileSelectiveAccessHandlerTest.java

```diff
@@ -188,4 +224,78 @@ private List<GetResult> createCaptureObjectsProfile2() {
 
     return Collections.singletonList(getResult);
   }
+
+  private List<GetResult> createPublicCaptureObjects() {
+    DataObjectDefinitions.getClockDefinition();
```

Comment:
The result of this function is not used. Shouldn't the clock object be added to the getResult?

Reply:
Removed the unused line. The first structureData was the clock definition. I have updated the naming to clarify it is the clock definition.